docs/spark-connect-overview.md

```diff
@@ -267,23 +258,17 @@ connection that is initialized at REPL startup.
 export SPARK_REMOTE="sc://myhost.com:443/;token=ABCDEFG"
 spark-connect-repl
 {% endhighlight %}
-or
-{% highlight bash %}
-SPARK_REMOTE="sc://myhost.com:443/;token=ABCDEFG" spark-connect-repl
-{% endhighlight %}
 
-#### Use CLI arguments
+or
 
-The customizations may also be passed in through CLI arguments as shown below:
 {% highlight bash %}
-spark-connect-repl --host myhost.com --port 443 --token ABCDEFG
+SPARK_REMOTE="sc://myhost.com:443/;token=ABCDEFG" spark-connect-repl
```

Comment:
should we replace `spark-connect-repl` with `spark-shell`?

Reply:
Yup